sos/plugins/apache.py

```diff
@@ -51,6 +51,14 @@ def setup(self):
             "/etc/httpd24/conf/httpd.conf",
             "/etc/httpd24/conf.d/*.conf",
             "/etc/httpd24/conf.modules.d/*.conf",
+            # Red Hat JBoss Core Services
+            "/opt/rh/jbcs-httpd24/root/etc/httpd/conf/httpd.conf",
+            "/opt/rh/jbcs-httpd24/root/etc/httpd/conf.d/*.conf",
+            "/opt/rh/jbcs-httpd24/root/etc/httpd/conf.modules.d/*.conf",
+            # Red Hat Software Collections
+            "/opt/rh/httpd24/root/etc/httpd/conf/httpd.conf",
+            "/opt/rh/httpd24/root/etc/httpd/conf.d/*.conf",
+            "/opt/rh/httpd24/root/etc/httpd/conf.modules.d/*.conf",
```

Comment:
These config files are similar to other's httpd config files. (It's just located under different diretory "/opt/rh" in order to avoid conflict to default one.) So, I think these files do not contain secret information that we should obfuscate.